nominations/templates/index.html

```diff
@@ -12,7 +12,9 @@
   <div class="row blue-bg nina-bg white-text pb4">
 
     <div class="container pt2">
-      {% nominations_platform_alert %}
+      <div class="col-md-12">
```

Comment:
There should always be a row in between container and column. Also `col-md-12` is the same as `col-xs-12` fyi.

Reply:
👍

Reply:
Actually rows aren't required, and I'd rather not rewrite this whole page to add rows.

Reply:
Or, rewording that, fixing this page to be proper rows and container structure is outside of scope here.

Reply:
I'm not sure what you mean. You don't need to fix the whole page. You just need to put `<div class="row">` in between the container and col divs. It should be a trivial change?

Bootstrap grid apparently supports using container without rows or columns. However for the grid it should be container > row > column. So can we please do one or the other?

Also would be nice to have conditional None handling if there is no alert, but that's probably out of scope.

Reply:
Since the rest of the page doesn't have rows here we need to do that on every container and column on this page or we'll have different padding in different sections. On top of that we have nested containers which aren't allowed in bootstrap so really we need a full reworking here. This is in line with how page has been built and isn't wrong per se so don't think it's worth spending time here fixing when already over on this ticket.

Reply:
👍